jobs/build/custom/Jenkinsfile

```diff
@@ -138,7 +138,7 @@ node {
         release = params.RELEASE.trim() ?: buildlib.defaultReleaseFor(params.BUILD_VERSION)
     }
     // If any arch is ready for GA, use signed repos for all (plashets will sign everything).
-    def out = buildlib.doozer("--group=openshift-${params.BUILD_VERSION} config:read-group --yaml release_state",
+    def out = buildlib.doozer("--group=openshift-${params.BUILD_VERSION} config:read-group --yaml software_lifecycle",
```

Comment:
The output before was
```
$ doozer -q -g openshift-4.13@155694be74cb2f020f6fafeaf6e4b3fba89646a2 --assembly stream config:read-group --yaml release_state
pre_release: []
release:
- x86_64
- ppc64le
- s390x
- aarch64
```

Vs now

```
$ doozer -q -g openshift-4.13 --assembly stream config:read-group --yaml software_lifecycle
phase: release
```

Since the outputs are different, we also need to change how the output's are used

Reply:
So `def repo_type = archReleaseStates['phase'] == 'release' ? 'signed' : 'unsigned'`
I think should work